RELEASE.md

```diff
@@ -1,4 +1,4 @@
-# (Unreleased)
+# Release 0.2.2
```

Comment:
In the line below:
* that for example that -> for example that
* rencers -> renders